src/tools/perf/perftest.c

```diff
@@ -169,7 +169,11 @@ ucs_status_t init_test_params(perftest_params_t *params)
     static const struct sockaddr_storage empty_addr = {};
 
     memset(params, 0, sizeof(*params));
-    params->super.api               = UCX_PERF_API_LAST;
+
+    /* UCT tests require specified device (-d) on both sides - client and server,
```

Comment:
the api should be detected by test code, maybe this is what the merge function should check?